bazel.el

```diff
@@ -88,8 +88,17 @@
           "https://github.com/bazelbuild/buildtools/tree/master/buildifier")
   :risky t)
 
-;;;; Commands to run Buildifier.
+(defvar bazel--magic-comment-rx
+  '(or "@unused"
```

Comment:
I guess it would be better to translate this into a string here (using `rx`) and then use `(regexp ...)` below in `rx-to-string`. That would shift the interpretation of the `rx` form to compile/load time.

Reply:
Done.